hazelcast/src/main/java/com/hazelcast/map/impl/query/MapQueryEngineImpl.java

```diff
@@ -526,6 +526,11 @@ protected void addResultsOfPagingPredicate(List<Future<QueryResult>> futures, Co
 
             Collection<Integer> tmpPartitionIds = queryResult.getPartitionIds();
```

Comment:
Yes, that's the variable's name in previous code but it also struck me as not-fitting so will fix it.